### PR TITLE
[greek_font] Update font for keyboard_info

### LINKED ISSUE #1704: galaxie_greek: default font on keymanweb.com is awful
So the ELTW people tell me

### PR DESCRIPTION
**Pull request by LornaSIL**
Fixes #1704 
Also adds a font to the sil_greek_polytonic web keyboard

### CODE REVIEW

**Review by DavidLRowe — APPROVED** [review]

**Comment by LornaSIL** [comment]
So far it looks like the keyboard layout is using Gentium, but the window where you type is still not Gentium.

**Comment by DavidLRowe** [comment]
@mcdurdin Can you check this fix for issue #1704 ?

File: release/g/galaxie_greek_positional/galaxie_greek_positional.keyboard_info [hunk 1]
```diff
@@ -6,15 +6,23 @@
         "keys": "e]davkrusen o[[ I]hsou~s ",
         "text": "ἐδάκρυσεν ὁ Ἰησοῦς ",
         "note": "\"Jesus wept\", John 11:35; note final sigma entered using s then spacebar"
-      }
+      },
+       "font": { 
+         "family": "Gentium Plus", 
+         "source": "GentiumPlus-Regular.ttf.ttf"
```
Comment (mcdurdin) [inline release/g/galaxie_greek_positional/galaxie_greek_positional.keyboard_info:12]: This has `.ttf` twice -- that's why it isn't working correctly 😸

File: release/g/galaxie_greek_positional/galaxie_greek_positional.keyboard_info [hunk 2]
```diff
@@ -6,15 +6,23 @@
...
     "el": {
       "example": {
         "keys": "edakrusen o Ihsous ",
         "text": "εδακρυσεν ο Ιησους ",
         "note": "\"Jesus wept\", John 11:35; note final sigma entered using s then spacebar"
-      }
+      },
+       "font": { 
+         "family": "Gentium Plus", 
+         "source": "GentiumPlus-Regular.ttf.ttf"
```
Comment (mcdurdin) [inline release/g/galaxie_greek_positional/galaxie_greek_positional.keyboard_info:23]: Also here!

File: release/sil/sil_greek_polytonic/sil_greek_polytonic.keyboard_info [hunk 3]
```diff
@@ -1,6 +1,19 @@
 {
     "license": "mit",
-    "languages": ["grc-Grek", "el"],
+  "languages": {
+    "grc-Grek": {
+       "font": { 
+         "family": "Gentium Plus", 
+         "source": "GentiumPlus-Regular.ttf.ttf"
```
Comment (mcdurdin) [inline release/sil/sil_greek_polytonic/sil_greek_polytonic.keyboard_info:7]: Ditto

File: release/sil/sil_greek_polytonic/sil_greek_polytonic.keyboard_info [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+    "grc-Grek": {
+       "font": { 
+         "family": "Gentium Plus", 
+         "source": "GentiumPlus-Regular.ttf.ttf" 
+    }
+    },
+    "el": {
+       "font": { 
+         "family": "Gentium Plus", 
+         "source": "GentiumPlus-Regular.ttf.ttf"
```
Comment (mcdurdin) [inline release/sil/sil_greek_polytonic/sil_greek_polytonic.keyboard_info:13]: One last one!

**Comment by DavidLRowe** [comment]
@mcdurdin So obvious once you point it out! Thanks.

**Pull request merged**